wgengine/ifstatus.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
How about we call this file `waitiface_noop.go` instead. and the other one `waitiface_windows.go`?